backend/Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-ruby file: ".ruby-version"
+ruby "3.2.3"
```

Comment:
@compwron I note this change was made specifically here: https://github.com/rubyforgood/Flaredown/commit/ae106635556f7be463e3a708383bacda621ccecf - do you see any harm in changing it to be a hard-coded version?

The reason I did this was that I was getting the following error in docker and this was the easiest fix to unblock me
<img width="836" alt="Screenshot 2024-04-18 at 11 41 19" src="https://github.com/rubyforgood/Flaredown/assets/21108297/c3905393-456b-4eb6-bcd1-6996aee6e8cb">

Reply:
Do we need to add the `.ruby-version` file to the Dockerfile context?

Reply:
I don't think so. And personally i like hardcoding it too -- .ruby-version is really for your editor and maybe its better to globally git ignore your preferred toolset (I like asdf for example, so i get .tool-versions) 

Gemfile will stop and give the right error message if your Ruby is wrong version and changing ruby to ME is something you just _do_

Reply:
Ah yes - I believe you're right in terms of fixing the error screenshot I posted. 
Saying that, I don't feel strongly either way whether this should be hard-coded or linked to the `.ruby-version` file. I do agree however that it can get confusing having multiple files determining versions. 
I personally use `.tool-versions`, but I think the dockerisation can be agnostic of this.